bugbug/model.py

```diff
@@ -69,14 +91,11 @@ def bugs():
             explainer = shap.TreeExplainer(self.clf)
             shap_values = explainer.shap_values(X_train)
 
-            print('Feature ranking (top 20 features):')
-            # Calculate the values that represent the fraction of the model output variability attributable
-            # to each feature across the whole dataset.
-            shap_sums = np.abs(shap_values).sum(0)
-            rel_shap_sums = shap_sums / shap_sums.sum()
-            indices = np.argsort(rel_shap_sums)[::-1][:20]
-            for i, index in enumerate(indices):
-                print(f'{i + 1}. \'{feature_names[index]}\' ({rel_shap_sums[index]})')
+            important_features = self.get_important_features(importance_cutoff, shap_values)
+
+            print(f'\nTop {len(important_features)} Features :')
+            for i, [importance, index, _unused] in enumerate(important_features):
+                    print(f'{i + 1}. \'{feature_names[int(index)]}\' ({importance})')
```

Comment:
Let's show whether the feature contribution is positive or negative here too, like in `classify`.